mkdocs.yml

```diff
@@ -99,6 +104,7 @@ extra:
 
 nav:
   - Frontpage: index.md
+  - Installation: installation.md
```

Comment:
Moved as I think this makes more sense here.

Reply:
The getting started section in the front page leads directly to the getting started chapter.

If you want to put the installation chapter in between, we should rethink the getting started chapter as well.